hack/istio/install-scale-mesh-demo.sh

```diff
@@ -52,7 +54,9 @@ install_scale_mesh_demo() {
     else
       ${CLIENT_EXE} create ns depth-${x}
     fi
-    ${CLIENT_EXE} label namespace  depth-${x} istio-injection=enabled --overwrite=true
+    if [ "${AUTO_INJECTION}" != "" ]; then
```

Comment:
Shouldn't this be explicitly checking for true - first, to be consistent with all the other ones you added in the other scripts, and second, because this will inject sidecars even if someone sets the value to `"false"`:
```suggestion
    if [ "${AUTO_INJECTION}" == "true" ]; then
```